tests/unit/s3routes/routesUtils/isValidObjectKey.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+const routesUtils = require('../../../../lib/s3routes/routesUtils.js');
+
+const bannedStr = 'banned';
+const prefixBlacklist = [];
+
+describe('routesUtils.isValidObjectKey', () => {
+    it('should return isValid false if object key name starts with a ' +
+    'blacklisted prefix', () => {
+        const result = routesUtils.isValidObjectKey('bannedkey', [bannedStr]);
+        // return { isValid: false, invalidPrefix };
+        assert.strictEqual(result.isValid, false);
+        assert.strictEqual(result.invalidPrefix, bannedStr);
+    });
+
+    it('should return isValid false if object key name exceeds length of 915',
+    () => {
+        const key = 'a'.repeat(916);
+        const result = routesUtils.isValidObjectKey(key, prefixBlacklist);
+        assert.strictEqual(result.isValid, false);
+    });
```

Comment:
Could be useful to add add a UTF-8 character unit test case that meets the key limit after the bucket prefix is applied (when using a max-length bucket name).